client/src/components/LanguageOptions.js

```diff
@@ -1,25 +1,13 @@
 const languageOptions = [
-  { key: "Ar", text: "Arabic", value: "Arabic" },
-  { key: "Chi", text: "Chinese", value: "Chinese" },
-  { key: "Dan", text: "Danish", value: "Danish" },
-  { key: "Dut", text: "Dutch", value: "Dutch" },
-  { key: "En", text: "English", value: "English" },
-  { key: "Fre", text: "French", value: "French" },
-  { key: "Ger", text: "German", value: "German" },
-  { key: "Grek", text: "Greek", value: "Greek" },
-  { key: "Hun", text: "Hungarian", value: "Hungarian" },
-  { key: "Ita", text: "Italian", value: "Italian" },
-  { key: "Jap", text: "Japanese", value: "Japanese" },
-  { key: "Kor", text: "Korean", value: "Korean" },
-  { key: "Lith", text: "Lithuanian", value: "Lithuanian" },
-  { key: "Per", text: "Persian", value: "Persian" },
-  { key: "Pol", text: "Polish", value: "Polish" },
-  { key: "Por", text: "Portuguese", value: "Portuguese" },
-  { key: "Rus", text: "Russian", value: "Russian" },
-  { key: "Spa", text: "Spanish", value: "Spanish" },
-  { key: "Swe", text: "Swedish", value: "Swedish" },
-  { key: "Tur", text: "Turkish", value: "Turkish" },
-  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+  { text: " Mandarin Chinese", value: "zh-cn" },
```

Comment:
Still a problem: remove the space at the beginning :)

Reply:
I removed the space